{{cookiecutter.app_name}}/Makefile

```
@@ -1,4 +1,7 @@
-.PHONY: dev
+.PHONY: build-dependencies dev
+
+build-dependencies:
```

I recommend to follow what Hoang did https://github.com/hoangmirs/go-scraper/blob/6775880d3c17f93150d8274a1f2e8edce824ad35/Makefile#L3-L6

```suggestion
install-dependencies:
	go get github.com/cosmtrek/air@v1.15.1
	go mod tidy
```

What about `forego`, when will it be added?

---

Updated command in 94856bc678b3273898e22021f62b1f82cb76299c.

I decide not to add `forego` or `goreman` in this release because we have only 1 process and also I plan to deploy to Heroku using docker so we don't need Procfile.
I want to keep only necessary thing and keep it simple as possible.

What do you think?

---

I agree then not to include it for 1.0.0 to keep it focused. However, I do think that `forego` should be added by default in the next version as having more than one process is very common. How about adding a new Github Project for `2.0.0` and start adding this kind of "wished" feature to that board? This can help future developers to know the roadmap for the next version as we are now making critical choices.

---

Sounds great. I will create the 2.0.0 project.

---

Created issue in https://github.com/nimblehq/gin-templates/projects/2